### DISCUSSION TITLE
Downloading different language

### DISCUSSION THREAD

**Discussion by kindlerl** [aclap-dev/video-downloadhelper#1005] [Help and Q&A]
### Your question

I am on a Mac, OS X 11.7.10, using the Firefox browser and the latest Video Downloadhelper.  On a particular site, I pull up the video and start playing it in English, then I select the Video Downloadhelper icon to open the panel, then select the Download button for the desired resolution.  The video will download, but it will download a version in a different audio language.  I cannot see any way to configure VDH to force a language.

Anyone else experience this?

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"6266d0bb44c140b4"}
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:128.0) Gecko/20100101 Firefox/128.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/Users/richardkindle/Downloads"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 4,
    "successfull_dl": 13
}

**Comment by GrampaWildWilly** [comment]
You should have been able to find these yourself:
#956
#973
#985

**Comment by johnmuir** [comment]
Yes, me too as they say, and the offhand reply from GrampaWildWilly doesn't really help anyone not versed in the intricacies of video. There really ought to be a more reasonable solution for normal users. Did you happen to find one?

**Reply by kindlerl** [reply]
I did not.

**Reply by johnmuir** [reply]
On another thread jcfrog who seems to be a maintainer said "It's a known issue, the dev team is working on it." So we can but hope ;-)

**Reply by jcfrog** [reply]
It's in the todo list, sorry we have a lot of issues these days with Youtube.

**Reply by johnmuir** [reply]
Thank you for your work!

**Reply by adlerscout** [reply]
any update on this? I just noticed this on a YouTube video I downloaded

**Reply by jcfrog** [reply]
Not yet, sorry.

**Reply by adlerscout** [reply]
Thank you for the prompt reply. Isn't there anyway to address this. The original video is in english, my YouTube settings are all in english. When I play the video, it plays in english, but when I download it, I get it in german.

**Reply by jcfrog** [reply]
We know, sorry, Firefox v10 should fix that but still needs time to be published.

**Reply by adlerscout** [reply]
Which version do you mean with v10? it's not on 143...

**Reply by jcfrog** [reply]
Video DownloadHelper for Firefox is still on V9 and will be soon updated to V10 like Chrome and Edge extensions.

**Comment by desolate-master** [comment]
is there a fix for this yet it seems to be a year later and still we are getting foreign language downloads on youtube (or is this a way youtube is now blocking downloads?)

**Reply by jcfrog** [reply]
The V9.5.0.2 update fixed the issue temporarily. The upcoming V10 update should be more effective.

**Comment by jillmj** [comment]
I'm using 9.5.0.2 on Firefox/Linux Mint and today for the first time, I am getting an English UK video downloading with a German soundtrack. All my settings are set correctly for language, and subtitles with UK english. Happens whether I am signed in to YouTube or not. Happens with every format/definition available. Today's video is an English one, and even the subtitles are set to English. But all the downloads, including a separate audio download, have a german soundtrack.
I've tried with the later MS Edge VDLH extension, and managed to get an English version.

**Reply by TerryDytl** [reply]
Same problem here with English on screen and German on recording

**Comment by xPaPaDx** [comment]
Doesn't matter what machine or OS I'm using. Linux, Mac, Windows 10 (a few days ago) and now Windows 11 I get the German voice over when it should be English. Latest build/update etc.

**Reply by Mattmjs** [reply]
It's not OS related, the extension needs an update. When that arrives I wouldn't know ( user like you ).
If you're desperate and don't want to wait, search a youtube download site that will provide it in English.

**Reply by jillmj** [reply]
> Doesn't matter what machine or OS I'm using. Linux, Mac, Windows 10 (a few days ago) and now Windows 11 I get the German voice over when it should be English. Latest build/update etc.

See my earlier post - by using MS-Edge browser's VDLH extension I got a later extension - version 10 -which downloaded an English version of the video I wanted. Check the extension version number (not the browser version). The Edge version is v10 as is the Chrome version. My normal firefox extension was 9.5.0.2

**Reply by Antontwo** [reply]
Just downloaded a piano tutor in English and ended up with a German version. Don't understand it at all.

**Reply by Mattmjs** [reply]
Install the latest version 10 and it should be in English.

**Reply by Antontwo** [reply]
Thanks Mattmjs. Where do I go to find the latest version 10?

**Reply by Antontwo** [reply]
I just checked in Firefox add-on and my VDH is version 9.5.0.0 (BETA it says when I hover over the application at the top of my screen). I clicked on a gearwheel for updates and it says I have then latest version, so what do you mean by version 10, Mattmjs?

**Reply by Antontwo** [reply]
I went to the add-ons page (which says version 9.5.0.0) so I went to the download homepage, and when I hover in the tab in my browser it says "v10.downloaderhelper.net" but I can't see anywhere on the page where I can download any v10!

**Reply by Antontwo** [reply]
I went to the older versions sections and there I see: v10.1.37.2 for January 2026. And I managed to download it. Yes, the video is in the English language now, thanks. The only problem is my download folder has changed from the external harddisk I was using and when I click the cogwheel and when I click on change the browser directory and I am just directed to the Firefox settings whereas before I could specify a folder on an external hard disk. Any idea how I can do that again, please? And why did I not get automatic updates, as my options (default, on, off) are set to default?

**Reply by Mattmjs** [reply]
> The only problem is my download folder has changed from the external harddisk

Yes because now that you're on version 10, it works on it's own. It doesn't use the companion app anymore.
Refer to this discussion for more information [sub directories downloads](https://github.com/aclap-dev/video-downloadhelper/discussions/2889)

You were on the beta channel and appeared to be the last version of it. There is the stable version from Mozilla add-ons.
There is now a rapid release version which replaces the beta from download helper site.

**Reply by Antontwo** [reply]
Thanks for going to all this trouble, Mattmjs. I'll have a look at that. Do you think I should change "default" update to "automatic" or doesn't it make any difference?

**Reply by Antontwo** [reply]
I've found the Download subdirectory in settings and it says "MyVideos/.

What do I do there to direct it to my external hard disk?

**Reply by Mattmjs** [reply]
Did you read the discussion I linked to. Someone mentions junctions and symbolic links. You add a sub folder to the downloads folder. Then you need some way to direct what goes there to another location. You'll will have to research more on your own.

**Reply by Antontwo** [reply]
Thanks a lot.

**Comment by macaaaron** [comment]
Hello, same for me.

I want to download French videos, but I got them with the YouTube AI English voice over.
I need to re-download them with JDownloader2.
It's very frustrating. 

I am on macOS 15.7.1, Firefox 144.0 and Video DownloadHelper Version 9.5.0.2 (stable).
